temporian/core/data/node.py

```diff
@@ -502,18 +502,19 @@ def input_node(
     Usage example:
 
         ```
-        # Without index
-        a = source_node(features=[("f1", tp.float64), ("f2", tp.string)])
+        >>> # Without index
+        >>> a = input_node(features=[("f1", tp.float64), ("f2", tp.str_)])
```

Comment:
tp.input_node?

Or is "tp" already imported?

Reply:
good catch. `input_node()` works here because we're on the module that defines it, but I'd agree it's best to use `tp.input_node` for consistency.